test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetWhyTests.cs

```diff
@@ -188,40 +188,6 @@ public void WhyCommand_EmptyPackageArgument_Fails()
             Assert.Contains($"Required argument missing for command: 'why'.", result.Errors);
         }
 
-        [Fact]
-        public async Task WhyCommand_InvalidFrameworksOption_WarnsCorrectly()
```

Comment:
This tests duplicates a NuGet.XPlat.FuncTest scenario exactly, and even has the same name: https://github.com/NuGet/NuGet.Client/blob/3050c87be5e223e7fed30e6eb6933a17d9d6212f/test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatWhyTests.cs#L223

FWIW, the last time these tests ran on the dev branch, the dotnet integration test took 2.75 seconds to run, the FuncTest took 87 milliseconds.